### PR TITLE
Display development, user, and organization data

### PR DESCRIPTION
**Pull request by beechnut**
- Updates gems
- Changes test passwords to use the right constant
- Fixes pointy-labeled buttons.

__Developments__

- Include, format, and style data
- Partial for rendering data table rows

__Organizations__

- Styles organization profile, adding a Gravatar
- Turns the Settings tab into an organization/edit form.

__Users__

- Styles profile, memberships.
- Update presentation of user/edit.

### CODE REVIEW

**Comment by beechnut** [comment]
Please note that I broke A LOT of tests.

**Comment by beechnut** [comment]
@allthesignals Would you give this a :+1:/:-1:? I'm happy handling the merge conflicts.

File: app/presenters/development_presenter.rb [hunk 1]
```diff
@@ -105,4 +111,18 @@ def short_address
...
+    item.attributes.select { |k, v|
+      categorized_attributes.fetch(category, {}).include?(k.to_sym) && !v.nil?
+    }
+  end
+
+  def categorized_attributes
+    { physical:   [:tothu, :commsf, :prjarea, :stories, :height],
+      housing:    [:singfamhu, :twnhsmmult, :lgmultifam, :tothu],
+      commercial: [:fa_ret, :fa_ofcmd, :fa_indmf, :fa_whs, :fa_rnd,
+        :fa_edinst, :fa_other, :fa_hotel] }
```
Comment (allthesignals) [inline app/presenters/development_presenter.rb]: Hard-coded groupings can be dangerous -- might consider refactoring later when there's time.

**Comment by allthesignals** [comment]
@beechnut :+1:

File: app/presenters/development_presenter.rb (hunk 1 above)
Comment (beechnut) [inline app/presenters/development_presenter.rb]: Agreed wholeheartedly :+1: :100: . I tried to find so many ways around this but ended up overengineering every time.

**Pull request merged**